launch/dynamo-run/src/subprocess/vllm_inc.py

```diff
@@ -59,9 +61,9 @@ def __init__(self, engine, default_sampling_params):
         self.default_sampling_params = default_sampling_params
 
     async def generate(self, request):
-        request_id = "1"  # hello_world example only
+        # logging.debug(f"Received request: {request}")
+        request_id = str(uuid.uuid4().hex)
```

Comment:
Just curious - why `.hex`? I don't think we do that anywhere else, just the `uuid4()` usually

Reply:
It's what vllm does to set it's request_id: https://github.com/vllm-project/vllm/blob/main/vllm/utils.py#L489-L490

I checked their commits and previously they did `str(uuid.uuid4().hex[:8])` so probably they dropped the slice but kept the hex, without it being very intentional

Reply:
TIL 👍

Reply:
The `.hex` will return it as a string without the hypens (`-`), so just more compact. It is also redundant to call `str(` on it again in this case.